src/css-image-value.js

```diff
@@ -0,0 +1,39 @@
+(function(internal, scope) {
+
+  function CSSImageValue(image) {
+    if (!(image instanceof Image)) {
```

Comment:
Do I need to make the from() function? Or just make the one like https://github.com/css-typed-om/typed-om/pull/135/files#diff-0c45aef1150ce960e1a54ed56eddadcbR132 (from line 132 to 142)?